app/assets/javascripts/app/views/hovercard_view.js

```diff
@@ -94,14 +95,20 @@ app.views.Hovercard = app.views.Base.extend({
 
   _populateHovercard: function() {
     var href = this.href();
-    href += "/hovercard.json";
+    href += ".json";
```

Comment:
Because `HovercardPresenter` is a presenter for the person model. I feel it's more presice to make a hovercard method for the `PersonPresenter` rather than having a whole separate presenter. Hovercard data is just another presentation for person data, and not some separate instance.

Also the hovercard presenter is not a child of BasePresenter. So even if leave it, it should be refactored heavily. That what I did in the first place, but then I changed my mind.